stdlib/public/Concurrency/Task.swift

```diff
@@ -496,10 +491,13 @@ func _runAsyncHandler(operation: @escaping () async -> ()) {
 
 @available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
 extension Task {
-  /// Suspends the current task for _at least_ the given duration
-  /// in nanoseconds.
+  /// Suspends the current task,
+  /// and wait for at least the given duration before resuming it.
```

Comment:
can we bring back the emphasis on "_at least_"? It's important to highlight that it's not an exact sleep time guarantee

Reply:
I need to check editorial guidelines, but I think we avoid emphasis in abstracts because they show up on their own in places like navigation and Xcode quick help.